### ISSUE TITLE
Parsing language names

### ISSUE THREAD

**Issue by rsprouse** [levmichael/saphon#72]
In the [Tupian nasal typology input spreadsheet](https://docs.google.com/spreadsheets/d/1dvXFvLIV4y84CglgjAl-ZVb09IuGazs1SzFO_UJpmnI) some values for `Language` contain strings with the language name and ISO 639-3 values, e.g. `Araweté [awt]`, which is parsed as the `name` and `iso_codes` values in the .json data format.

Other languages have a third part of the string. What is the intended meaning of the third part and how should it be parsed? Examples: `Avá-Canoeiro Goiás [avv-gos] (avá-canoeiro goiás)` and `Avá-Canoeiro [avv-tct] (avá-canoeiro tocantins)`.

**Comment by levmichael** [comment]
The languages that have a third part to the string are languages that require a dialect specification. More generally I agree with the point that there needs to a bunch of hand correction done. What would probably be ideal, actually, is to switch over to glottocodes, and get Harald Hammarstöm to add dialects for us. @mlapier What do you think?

**Comment by mlapier** [comment]
Yes, I agree. This is exactly why we were thinking of switching them all to
Glottocodes.

Myriam Lapierre
Assistant Professor
Department of Linguistics
University of Washington

*I recognize that the University of Washington stands on the lands and
shared waters of the Coast Salish Peoples; Duwamish, Puyallup, Suquamish,
Tulalip and Muckleshoot nations.*
